timesketch/frontend-v3/src/router/index.js

```diff
@@ -12,6 +12,7 @@ import Default from "@/layouts/Default";
 
 // Import App views
 import Home from "@/views/Home.vue";
+import Overview from "@/views/Overview.vue";
```

Comment:
What's the idea with the "Overview" page? :) Is this part of the "Sketch" view?

Reply:
I think I accidentally took that over from frontend-ng version:

```
 {
        path: '',
        name: 'Overview',
        redirect: { name: 'Explore' },
      },
```

I guess we don't need the Overview component and replace it with Canvas. I'll remove it from this CL as it is irrelevant for it anyway.